Runtime/Conditions/PositionalConditions/TeleportCondition.cs

```diff
@@ -0,0 +1,98 @@
+using System.Runtime.Serialization;
+using Innoactive.Creator.Core.Attributes;
+using Innoactive.Creator.Core.Properties;
+using Innoactive.Creator.Core.SceneObjects;
+using Innoactive.Creator.Core.Utils;
+using Innoactive.Creator.Core.Validation;
+
+namespace Innoactive.Creator.Core.Conditions
+{
+    /// <summary>
+    /// Condition which is completed when an 'XR Rig' gets teleported into the referenced <see cref="ITeleportProperty"/>.
```

Comment:
Does not have to be an XR Rig, could also be something else, like a regular camera or trainee object

Reply:
Same in the ITeleportProperty, but those are just details